.github/workflows/wine-arch.yml

```
@@ -0,0 +1,29 @@
+name: Wine Archlinux CI
```

```suggestion
name: Wine Arch Linux CI
```
See [here](https://wiki.archlinux.org/title/Arch_terminology#Arch_Linux).